src/plots/cartesian/layout_attributes.js

```diff
@@ -278,22 +280,19 @@ module.exports = {
 
         pattern: {
             valType: 'enumerated',
-            // TODO could add '%H:%M:%S'
-            values: ['%w', '%H', ''],
+            values: [DAY_OF_WEEK, HOUR, ''],
             dflt: '',
             role: 'info',
             editType: 'calc',
             description: [
                 'Determines a pattern on the time line that generates breaks.',
-                'If *%w* - Sunday-based weekday as a decimal number [0, 6].',
-                'If *%H* - hour (24-hour clock) as a decimal number [0, 23].',
-                'These are the same directive as in `tickformat`, see',
-                'https://github.com/d3/d3-time-format#locale_format',
+                'If *' + DAY_OF_WEEK + '* - Sunday-based weekday as a decimal number [0, 6].',
+                'If *' + HOUR + '* - hour (24-hour clock) as decimal numbers between 0 and 24.',
```

Comment:
Oh we should note here that weekday needs to be an integer but hour can be fractional.